systemtest/src/test/java/io/strimzi/systemtest/security/OpaIntegrationST.java

```diff
@@ -31,12 +31,14 @@
 
 import java.io.IOException;
 
+import static io.strimzi.systemtest.Constants.ARM_UNSUPPORTED;
 import static io.strimzi.systemtest.Constants.INTERNAL_CLIENTS_USED;
 import static io.strimzi.systemtest.Constants.REGRESSION;
 import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
 
 @Tag(REGRESSION)
 @Tag(INTERNAL_CLIENTS_USED)
+@Tag(ARM_UNSUPPORTED)
```

Comment:
Why is this not supported? Does for example https://hub.docker.com/layers/openpolicyagent/opa/latest-static/images/sha256-7c754dc6ac0c9cd8309c9d90f43515c848f61a817d93b97128441ffff01110f3?context=explore not work?

Reply:
you are right, fixed